flight_phase_files/HALO_RF13_20200213_info.yaml

```diff
@@ -24,59 +24,112 @@ remarks:
 - During the whole flight precipitating cells were observed.
 - On the way to NTAS a few cold pools were observed with shallow sugar-type convection.
 segments:
-- kind: circle
+- kinds:
+  - circle
   name: c1
-  start: 2020-02-13 08:20:47
-  end: 2020-02-13 09:16:21
-- kind: circle_break
+  irregularities: []
+  segment_id: HALO-0213_c1
+  start: 2020-02-13 08:19:47
+  end: 2020-02-13 09:16:39
```

Comment:
Circle goes over more than 360 ° in heading, cut off some period at the end so that there is no overlap in the zoom on the circle ends plot.